mobile/src/calendar/event-dialog/change-vacation-end-date-event-dialog.tsx

```diff
@@ -86,7 +86,10 @@ const mapDispatchToProps = (dispatch: Dispatch<EventDialogActions>): ChangeVacat
         calendarEvent: CalendarEvent,
         startDate: Moment,
         endDate: Moment
-    ) => { dispatch(confirmVacationChange(employeeId, calendarEvent, startDate, endDate)); },
+    ) => { 
+        dispatch(startProgress());
```

Comment:
rename it somehow? Not clear that 'startProgress' means.

Reply:
fixed